test/unit/background/message-ports-test.js

```diff
@@ -80,7 +83,6 @@ describe("background > message ports", () => {
     itemId = result.item.id;
 
     expect(result.item).to.deep.include(item);
-    expect(selfListener).to.have.callCount(0);
```

Comment:
Since the datastore broadcasts messages to relay events from the Logins API, there's no sender or self available at that time to refrain from sending the event back to "self"